dss/api/drs.py

```diff
@@ -0,0 +1,51 @@
+import requests
+
+from flask import jsonify, make_response, request
+
+from dss import dss_handler
+from dss.api.files import get_helper
+from dss.config import Replica
+from dss.util.version import datetime_from_timestamp
+
+
+@dss_handler
+def get_data_object(uuid: str):
+    """
+    Return a DRS data object dictionary for a given DSS file UUID and version.
+    If the file is already checked out, we can return a drs_object with a URL
+    immediately. Otherwise, we need to send the request through the /access
+    endpoint.
+    """
+    # TODO: version lookup
+    with app.test_request_context(f'/files/{uuid}', method=HEAD, headers=request.headers):
+        req = get_helper(uuid, replica=Replica['aws'], version=None)
+    if req.status_code != 302:
```

Comment:
does there need to be any changes from a DSS response for a `301` to what DRS is expecting?